### PR TITLE
Streamline add friend functionality

### PR DESCRIPTION
**Pull request by davidchan1219**
**What this PR does** 📖

**Which issue(s) this PR fixes** 🔨

Fixes #
Updated Streamline add friend functionality
**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: layouts/chat.vue [hunk 1]
```diff
@@ -104,7 +107,7 @@ export default Vue.extend({
             (friend) => friend.address === this.$route.params.address
           )
       return recipient
-    },
+    }
```
Comment (Jekrimo) [inline layouts/chat.vue:110]: put trailing , back in

File: layouts/chat.vue [hunk 2]
```diff
@@ -117,7 +120,7 @@ export default Vue.extend({
     }
     window.addEventListener('resize', appHeight)
     appHeight()
-  },
+  }
```
Comment (Jekrimo) [inline layouts/chat.vue:123]: put trailing , back in

File: store/friends/actions.ts [hunk 3]
```diff
@@ -152,10 +152,9 @@ export default {
         'removeOutgoingRequest',
         friendAccountToOutgoingRequest(friendAccount).requestId
       )
-      return
+      return;
```
Comment (Jekrimo) [inline store/friends/actions.ts:155]: remove ;

Literally just had Matt on my review asking for these haha, good job David!

**Review by Jekrimo — CHANGES_REQUESTED** [review]

File: .eslintrc.js [hunk 4]
```diff
@@ -16,6 +16,7 @@ module.exports = {
     'vue/script-setup-uses-vars': 0,
     'import/named': 0,
     'standard/no-callback-literal': 0,
+    'comma-dangle': ['error', 'never'],
```
Comment (WanderingHogan) [inline .eslintrc.js:19]: please take this out - I believe @stavares843 took it out a few days ago

File: package.json [hunk 5]
```diff
@@ -126,7 +126,10 @@
     "jest": "^26.6.3",
     "lint-staged": ">=10",
     "prettier": "^2.2.1",
+<<<<<<< HEAD
     "simple-peer": "^9.11.0",
+=======
```
Comment (WanderingHogan) [inline package.json:131]: It looks like you might not have finished a rebase?

File: jest.config.js [hunk 6]
```diff
@@ -30,6 +30,11 @@ module.exports = {
   transformIgnorePatterns: [
     '/node_modules/',
     'node_modules/(?!@mylibrary/)',
+<<<<<<< HEAD
     '^.+\\.module\\.(css|sass|scss)$'
```
Comment (WanderingHogan) [inline jest.config.js:34]: another rebase problem it looks like?

File: components/tailored/friends/add/Add.vue [hunk 7]
```diff
@@ -29,6 +30,14 @@ export default Vue.extend({
     }
   },
   methods: {
+    _searchFriend: _.debounce(async function(this:any) {
+      if(this.accountID.length >= 40) {
+        await this.searchFriend()
+      } else {
+        this.error = this.$t('friends.invalid_characters') as string
```
Comment (WanderingHogan) [inline components/tailored/friends/add/Add.vue:37]: prefer guard clause over if...else

File: components/tailored/messaging/message/Message.html [hunk 8]
```diff
@@ -27,10 +27,18 @@
       :setReplyChatbarContent="setReplyChatbarContent"
       :emojiReaction="emojiReaction"
       :editMessage="editMessage"
-      :message="message"
+      :messageOwner="this.$props.message.from"
```
Comment (WanderingHogan) [inline components/tailored/messaging/message/Message.html:30]: can this just be `message.from` since the props are available in the template

**Review by WanderingHogan — CHANGES_REQUESTED** [review]
Looks like this PR might have had some issues while rebasing

File: .eslintrc.js (hunk 4 above)
Comment (stavares843) [reply .eslintrc.js:19]: correct yes

**Comment by davidchan1219** [comment]
fixed conflict and merged with dev branch.

**Comment by WanderingHogan** [comment]
A couple things - 

This looks like it is meant to address a few different tickets. Can you make 1 pull request per ticket. I see ap-158 here and i believe ap-151? Multiple tickets in a single ticket are okay if it's a very minor update to fix something, but these are two pretty big totally independent tickets

Please fill out the commit message with what the PR does, what ticket it addresses, notes, etc

When you are updating your branch to address conflicts, can you rebase your changes on top of dev instead of merging? we would like to see as few files changed and as few commits as possible to make the changes, it makes reviewing a lot easier.

Jeff and I made a few requests in the github ticket for changes, that i don't see updated, either.

**Comment by stavares843** [comment]
the fixes from this PR were added on separate PR's

**Pull request closed without merging**